compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -1616,17 +1616,6 @@ extern "C" void LLVMRustSetLinkage(LLVMValueRef V,
   LLVMSetLinkage(V, fromRust(RustLinkage));
 }
 
-// FIXME: replace with LLVMConstInBoundsGEP2 when bumped minimal version to llvm-14
```

Comment:
At time when i placed this comment i didn't realized that current minimum llvm was 14. Is there easy to find places where it noted?

Reply:
The *easiest* thing to see is the PR job that runs on the minimum, currently `x86_64-gnu-llvm-14`.

We also mention updates in RELEASES.md, and the actual enforcement is here:
https://github.com/rust-lang/rust/blob/fd68a6ded951bd7b852ab8107007f7145e3ad6ec/src/bootstrap/llvm.rs#L519-L533